app/ui/legacy/src/main/java/com/fsck/k9/view/RecipientSelectView.java

```diff
@@ -61,6 +62,7 @@ public class RecipientSelectView extends TokenCompleteTextView<Recipient> implem
     private static final int LOADER_ID_ALTERNATES = 1;
 
 
+    private EmailAddressValidator validator = new EmailAddressValidator();
```

Comment:
Could you please change this to depdency injection and it could also be set to final?

```suggestion
    private final EmailAddressValidator validator = DI.get(EmailAddressValidator.class);
```